pkg/reconciler/serverlessservice/global_resync_test.go

```diff
@@ -59,6 +59,7 @@ func TestGlobalResyncOnActivatorChange(t *testing.T) {
 	grp := errgroup.Group{}
 	defer func() {
 		cancel()
+		tcancel()
```

Comment:
```suggestion
		cancel()
```

Delete line 58 which has `
ctx, cancel := context.WithCancel(ctx)
`